Lekce-8-HW/Truck.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Lekce_8_HW
+{
+    internal class Truck : Vehicle, IDriveable
+    {
+        public int NakladniProstor { get; set; }
+        public override void UkazInfo()
+        {
+            Console.WriteLine("Nakladak " + Brand + "s najezdem " + Mileage + ". Ma objem " + NakladniProstor + " metru krychlovych.");
+        }
+        public void Drive()
+        {
+            Console.WriteLine("Nakladak " + Brand + " jede pryc.");
+        }
+        public void NalozNaklad(int weight)
```

Comment:
Tohle vypada jako naznak na vypracovani bonusove casti hw, ale pak v `Program.cs` nevidim, ze bys ty Trucky rozeznal a zavolal tu jejich `NalozNaklad` metodu. Je to jen jeden if navic v podstate :) Kdyztak muzu poradit